src/customFileParser/sharedFileParserTypes.ts

```diff
@@ -42,7 +42,17 @@ interface UncompressedValues {
   fileId?: number[];
 }
 
-class NodeIdReader {
+export type PerSectorCompressedData = {
+  primitives: CompressedGeometryData[],
+  instancedMesh?: CompressedGeometryData,
+  mergedMesh?: CompressedGeometryData,
+};
+
+export type PerSectorCompressedDataDictionary = {
```

Comment:
Please don't use "Dictionary" in JavaScript/TypeScript, that's not a term in the language. I would suggest just putting `[path: string]: PerSectorCompressedData;` everywhere, it's not a big deal (and it's barely longer). IMO easier to work with as well, having a type calling "dictionary of X" isn't very informative, since it expresses exactly the same information as `[key]: X`, but it adds an extra layer of indirection when you want to look up what X is.

Reply:
Gotcha. Fixed